Resources/config/console.xml

```diff
@@ -16,6 +16,14 @@
             <argument type="collection" /> <!-- user providers -->
             <tag name="console.command" command="lexik:jwt:generate-token" />
         </service>
+
+        <service id="lexik_jwt_authentication.generate_keypair_command" class="Lexik\Bundle\JWTAuthenticationBundle\Command\GenerateKeyPairCommand" public="true">
+            <argument type="service" id="filesystem" />
+            <argument type="string">%env(default::resolve:JWT_SECRET_KEY)%</argument>
+            <argument type="string">%env(default::resolve:JWT_PUBLIC_KEY)%</argument>
+            <argument type="string">%env(default::JWT_PASSPHRASE)%</argument>
```

Comment:
Those arguments should be abstract and replaced in the DI extension (LexikJWTAuthenticationExtension) by their respective config values

Reply:
Well, that's what I tried at first, but I didn't figure out how to grab the private key path / the public key path / the passphrase.

If I put a `dump($config)` around line 69 of `LexikJWTAuthenticationExtension`, here's what I get:
<img width="688" alt="Capture d’écran 2020-12-20 à 17 30 34" src="https://user-images.githubusercontent.com/5569077/102718558-17db6700-42e9-11eb-800f-3595fcb0aaef.png">

Any idea?

Reply:
Yes, that's expected :)
Env parameters (`%env()%`) are resolved at runtime (which means they can be changed at anytime, the change will take effect on your services without clearing the cache). 
For this to work, Symfony needs to replace that `%env()%` strings by what we call "env placeholders" during the compilation phase, instead of resolving their actual value as for regular parameters (because runtime values are not known yet) so that one can manipulate such arguments in their DI extensions/compiler passes.

The test you added should be enough to make sure everything works.

Reply:
Alright, didn't know how it works behind the scenes 👍 
I just pushed a new commit, but it looks like I still miss something. Now when I run the command I get this:
<img width="694" alt="Capture d’écran 2020-12-20 à 18 18 30" src="https://user-images.githubusercontent.com/5569077/102719705-cc788700-42ef-11eb-864d-57f23f26f102.png">

Which seems to indicate that file paths are empty.

Besides, when I set arguments as `<argument type="abstract" />` in `console.xml`, I get the following error:

> Argument 2 of service "lexik_jwt_authentication.generate_keypair_command" is abstract.  
I probably miss something here as well, your help is very welcome :-)